src/main/java/net/openhft/chronicle/wire/VanillaMethodReader.java

```diff
@@ -414,7 +413,7 @@ else if (o2 != null)
 
     @NotNull
     protected MethodWireKey createWireKey(@NotNull Method m, String name) {
-        MethodId annotation = Annotations.getAnnotation(m, MethodId.class);
+        MethodId annotation = Jvm.findAnnotation(m, MethodId.class);
```

Comment:
none of these annotation changes are related to javadoc. Are they meant to be here?

Reply:
I will separate them into a Java21 improvement PR.